src/test/java/com/epam/tc/hw5/pages/DifferentElementsPageMainContainer.java

```diff
@@ -0,0 +1,57 @@
+package com.epam.tc.hw5.pages;
+
+import java.util.List;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+import org.openqa.selenium.support.PageFactory;
+import org.openqa.selenium.support.ui.Select;
+
+public class DifferentElementsPageMainContainer {
+
+    private String checkboxWater = "Water";
+    private String checkboxWind = "Wind";
+    private String radioSelen = "Selen";
+    private String colorYellow = "Yellow";
+
+    @FindBy(css = ".label-checkbox")
+    private List<WebElement> checkboxes;
+
+    @FindBy(css = ".label-radio")
+    private List<WebElement> radio;
+
+    @FindBy(css = "div.colors>.uui-form-element")
+    private WebElement colorDropdown;
+
+    @FindBy(css = ".panel-body-list.logs>li")
+    private List<WebElement> logEntries;
+
+    public DifferentElementsPageMainContainer(WebDriver driver) {
+        PageFactory.initElements(driver, this);
+    }
+
+    public DifferentElementsPageMainContainer selectCheckboxes() {
+        for (WebElement opt : checkboxes) {
+            if (opt.getText().equals(checkboxWater) || opt.getText().equals(checkboxWind)) {
```

Comment:
In future check please for streams, it would be really useful

Reply:
refactored selectCheckboxes() method with stream